apps/email/js/cards/message_list.js

```diff
@@ -231,6 +231,7 @@ return [
           model.element = node;
           node.message = model;
           this.updateMessageDom(true, model);
+          this.searchButton.disabled = false;
```

Comment:
We do not need to wait for the vscroll callback in this case, but just set disabled false as part of the normal createdCallback setup, maybe as an else case to line 180.